pyiron/lammps/interactive.py

```diff
@@ -539,7 +519,29 @@ def update_potential(self):
         self._interactive_lib_command(self.potential.Config[0][1])
 
     def interactive_indices_getter(self):
-        return super(LammpsInteractive, self).interactive_indices_getter().tolist()
```

Comment:
Can you test if these changes have an impact on the performance of the interactive mode? If that is the case it might make sense to check the calculation mode in the generic_input: `self._generic_input["calc_mode"] in ["vcsgc"]`

Reply:
I did two MD tests locally on my laptop, both 6750 Al atoms at room temperature.

| n_steps | n_print | updating | static |
|---------|---------|----------|--------|
| 1000    | 10      | 2m44s    | 2m35s  |
| 100     | 1       | 2m20s    | 2m22s  |

I'm comfortable leaving it 'always on'. 

Non-interactively this takes 17s, so interactive is still super slow, but with some thousands of atoms this doesn't seem to make a big difference.